src/NuGet.Core/NuGet.Packaging/Signing/Utility/SignatureUtility.cs

```diff
@@ -260,7 +260,9 @@ public static IX509CertificateChain GetCertificateChain(PrimarySignature primary
                 signingSpecifications,
                 includeChain);
         }
+#endif
 
+#if IS_SIGNING_SUPPORTED
```

Comment:
Remove lines 263 and 265 but leave 1 blank line between members.  I don't know why you're individually wrapping members with the same `#if...#endif` condition.

Reply:
Thanks! Fixed. I tried to enable as less as possible signing APIs so I enabled them individually. But I didn't do a "merge" afterwards.